kitchen.yml

```diff
@@ -73,6 +71,13 @@ platforms:
       run_command: /usr/lib/systemd/systemd
 
   ## SALT `2019.2`
+  - name: amazonlinux-1-2019-2-py2
+    driver:
+      image: amazonlinux:1
+      provision_command:
+        - curl -o bootstrap-salt.sh -L https://bootstrap.saltstack.com
+        - sh bootstrap-salt.sh -XdPfq -x python2 git 2019.2
```

Comment:
Sorry, I wanted to come back to this after doing some investigating - but by pulling via `git` here, we're pulling old code. Is this intentional or should it be `stable` instead?

Reply:
@dafyddj Thanks for bringing this up.  Just ran some comparisons and it isn't as clear cut as that:

Version | `git` | `stable`
:-: | --- | ---
**2019.2** | Latest commit [779bc17](https://github.com/saltstack/salt/tree/2019.2) on 25 Sep | Latest commit [ca2afa5](https://github.com/saltstack/salt/tree/v2019.2.2) 25 days ago
**2018.3** | Latest commit [db8cf2f](https://github.com/saltstack/salt/tree/2018.3) 10 days ago | Latest commit [d0cad3e](https://github.com/saltstack/salt/tree/v2018.3.4) on 7 Feb
**2017.7** | Latest commit [0b46284](https://github.com/saltstack/salt/tree/2017.7) on 23 Sep | Latest commit [d520f9a](https://github.com/saltstack/salt/tree/v2017.7.8) on 27 Aug 2018

The inconsistency probably surrounds the switch to the new branching scheme after `2019.2.2`.

So as long as my assumptions are correct, it looks like we need to use the following plan for the time being:

Version | Bootstrap
:-: | ---
**master** | `git master`
**2019.2** | `stable 2019.2`
**2018.3** | `git 2018.3`
**2017.7** | `git 2017.7`

There could be an argument to stick to `stable` to use the old releases rather than the latest branches but this is probably a good question to ask in the `wg`.

Reply:
Changes made after discussion in `wg`, using `stable` for all versions except `master` (obviously) -- for the time being!